src/db_api_util.ts

```diff
@@ -123,10 +301,37 @@ export async function logMessageToDb(
         databaseMessageEntry.archived,
       ]
     );
-
     logger.info(
       `DBAPIUTIL.logMessageToDb: Successfully inserted message into PostgreSQL database.`
     );
+
```

Comment:
I'd prefer to keep logMessageToDb's responsibility strictly to logging a message to the DB. Can we add this logic elsewhere, perhaps in a function called alongside where logMessageToDb is called?